### ISSUE TITLE
Header row out of kilter

### ISSUE THREAD

**Issue by DutchPete** [jmouka/flexdown-obsidian-plugin#1]
1<sup>st</sup> of all many thanks for this plug-in.

I made a layout with 5 columns and found the columns in the header row to be offset relative to the columns in the rows below.

I added `%% flex: 2`, which gave me an optimal result but still offset.

![image](https://user-images.githubusercontent.com/10794265/221501370-6ea6b47e-ee6e-4740-8893-f3bd86fe73e2.png)

**Comment by jmouka** [comment]
Hi, thanks for trying the plugin. Yes, this can happen because the flexbox model doesn't create a table, but a flexible column layout that will shrink/grow based on its content. 

You can render it as a table with "%% display: table" (https://github.com/jmouka/flexdown-obsidian-plugin#display-table) but that won't properly render the last row. 

You can also be explicit with the column widths and the shrink/grow rules, eg "%% flex-all: 1 1 100px" will set the initial width of each column to 100px and they will all grow/shrink at the same rate. You can also set the flex rule for each column individually by using "%% flex" with comma-separated values.

**Comment by DutchPete** [comment]
Thanks for coming back to me so quickly.

I forgot to mention, but I had tried the "%% display: table" option but that did not solve the “problem”, as you stated.

"%% flex-all: 1 1 100px" did the trick. Why do I need 2 ones before the 100px instead of just 1 one? I tried it with just 1 one and it does the same as with 2.

**Comment by jmouka** [comment]
Yeah, this is the challenge with the flexbox CSS model (https://developer.mozilla.org/en-US/docs/Web/CSS/flex), it can be a challenge, and I was worried people will find it confusing. The three values are: grow ratio, shrink ratio, and initial width, so value "1 1 100px" means all columns can grow at the same ratio, shrink at same ratio, and start off as 100px wide. There are quite a few ways of tweaking the column resizing rules. It gives you a lot of flexibility but can be confusing. The real goal of the flexdown plugin is to quickly make some columns, especially if the contents can be more complex, but not worry too much about getting it perfect, but also have a lot of flexibility. I think for your case the display: table would be a better solution if it was possible to specify column spans for specific cells. I'd like to make that change.. once I have a free moment ;)

**Comment by DutchPete** [comment]
I don't know why, but changing themes messed up the column alignment, and now even the one where it worked is misaligned.

I tried in the sandbox vault, and there it works perfectly, so it must be something to do with those themes.

I'll leave it as is, and wait for the `display: table` solution.
